storybook/stories/API/cartesian/Scatter.stories.tsx

```diff
@@ -465,49 +465,6 @@ const ReactiveProps = {
   },
 };
 
-const OtherProps = {
-  xAxis: {
-    table: {
-      category: 'Other',
-    },
-  },
-  yAxis: {
-    table: {
-      category: 'Other',
-    },
-  },
-  zAxis: {
-    table: {
-      category: 'Other',
-    },
-  },
-  top: {
-    table: {
-      category: 'Other',
-    },
-  },
-  left: {
-    table: {
-      category: 'Other',
-    },
-  },
-};
-
-const InternalProps = {
-  points: {
```

Comment:
people might mention that they use points to do some manual work but if they do that they can just use `Dot`

Reply:
I expect that whoever needs that amount of control might also render SVG directly. I would consider adding it back as an optional prop if there is strong demand.

Reply:
yep